pages/about.html

```diff
@@ -1,12 +1,104 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+<html lang="en" class="bg-night">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="../stylesheets/about-style.css">
+		<link rel="stylesheet" type="text/css" href="../stylesheets/style.css">
+		<title>About Me</title>
+
+	</head>
+
+	<body class="no-margin">
+		<div class="container">
+			<nav class="navbar white">
+				<div class="navbar-container">
+					<h1 class="no-margin"><a class="no-style" href="index.html">Hannah's Personal Website</a></h1>
+					<div class="menu-icon">
+						<input id="menu" class="no-margin" type="checkbox">
+						<label for="menu"></label>
+						<div class="hamburger-menu no-margin">
+							<ul class="menu-items">
+								<li><a class="no-style" href="index.html">HOME</a></li>
+								<li><a class="no-style" href="about.html">ABOUT ME</a></li>
+								<li><a class="no-style" href="portfolio.html">MY PORTFOLIO</a></li>
+								<li><a class="no-style" href="favorites.html">MY FAVORITES</a></li>
+							</ul>	
+						</div>
+					</div>
+				</div>
+			</nav>
+			<main class="main-block">
+				<div class="image-block">
+					<div class="my-image">
+						<img alt="profile picture" class="profile-image" src="../images/">
+					</div>
+					<img alt="outline circle" class="circle-outline" src="../images/hand-drawn-circle.png">			
+				</div>
+				<div class="profile-block white">
+					<div class="profile-title">
+						<div>My name is</div>
+						<div class="name">Hannah.</div>
+						<div class="profile-desc-1">
+							Seattle-transplant, former finance professional, aspiring software engineer.
+						</div>
+
+						<div class="socials">
+							<ul class="social-contacts">
+								<li><a class="no-style white" href="https://drive.google.com/file/d/1nPIhdCPWueGwTxfHZGI4UM7kRolsJfmP/view?usp=sharing">Resume</a></li>
+								<li><a class="no-style white" href="https://www.linkedin.com/in/hannah-tan/">LinkedIn</a></li>
+								<li><a class="no-style white" href="https://github.com/hannahtan96">Github</a></li>
+							</ul>
+						</div>
+					</div>
+				</div>
+				
+			</main>
+
+			<div class="profile-desc-2">
```

Comment:
There's a mix of semantic and div tags here that make it a bit difficult to read the HTML.  I recommend using semantic tags as much as possible, and then using div tags for formatting when needed.  For example, this text block would work well as an article or section tag.
```suggestion
			<article class="profile-desc-2">
```